src/integrationTest/java/org/opensearch/security/SnapshotSteps.java

```diff
@@ -0,0 +1,87 @@
+/*
+* Copyright OpenSearch Contributors
+* SPDX-License-Identifier: Apache-2.0
+*
+* The OpenSearch Contributors require contributions made to
+* this file be licensed under the Apache-2.0 license or a
+* compatible open source license.
+*
+*/
+package org.opensearch.security;
+
+import java.io.IOException;
+import java.util.Map;
+
+import org.awaitility.Awaitility;
+
+import org.opensearch.action.admin.cluster.repositories.delete.DeleteRepositoryRequest;
+import org.opensearch.action.admin.cluster.repositories.put.PutRepositoryRequest;
+import org.opensearch.action.admin.cluster.snapshots.create.CreateSnapshotRequest;
+import org.opensearch.action.admin.cluster.snapshots.create.CreateSnapshotResponse;
+import org.opensearch.action.admin.cluster.snapshots.delete.DeleteSnapshotRequest;
+import org.opensearch.action.admin.cluster.snapshots.get.GetSnapshotsRequest;
+import org.opensearch.action.admin.cluster.snapshots.get.GetSnapshotsResponse;
+import org.opensearch.action.admin.cluster.snapshots.restore.RestoreSnapshotRequest;
+import org.opensearch.action.admin.cluster.snapshots.restore.RestoreSnapshotResponse;
+import org.opensearch.client.RestHighLevelClient;
+import org.opensearch.client.SnapshotClient;
+import org.opensearch.snapshots.SnapshotInfo;
+import org.opensearch.snapshots.SnapshotState;
+
+import static java.util.Objects.requireNonNull;
+import static org.opensearch.client.RequestOptions.DEFAULT;
+
+class SnapshotSteps {
+
+	private final SnapshotClient snapshotClient;
+
+	public SnapshotSteps(RestHighLevelClient restHighLevelClient) {
+		this.snapshotClient = requireNonNull(restHighLevelClient, "Rest high level client is required.").snapshot();
+	}
+
+	//CS-SUPPRESS-ALL: It is not possible to use phrase "cluster manager" instead of master here
```

Comment:
If one doesn't exist, please create an issue for any suppressions https://github.com/opensearch-project/OpenSearch/issues/3663 applies for this case.

Reply:
I am not sure if I correctly understand your comment. The suppression is required because the class AcknowledgedResponse is placed in the package (org.opensearch.action.support.master
) which contains the word "master" in the name. The word "master" is prohibited by the Checkstyle rule https://github.com/opensearch-project/security/blob/main/checkstyle/sun_checks.xml#L200. In my opinion, the Checkstyle rule is too generic.

Reply:
https://github.com/opensearch-project/security/pull/2139#discussion_r986952050 captured this better - looks like this is resolved